proxy/http/HttpSM.cc

```diff
@@ -260,7 +260,104 @@ HttpVCTable::cleanup_all()
   }
 }
 
-#define SMDebug(tag, fmt, ...) SpecificDebug(debug_on, tag, "[%" PRId64 "] " fmt, sm_id, ##__VA_ARGS__)
+#if defined(Dbg) // Optimized debug available.
```

Comment:
Use of conditional compilation here means that we could backport this PR to a 9.x release without having to also backport https://github.com/apache/trafficserver/pull/8581 .  Could be useful if we need to backport later PRs that also change HttpSM.cc extensively.